modules/ardrivo/src/Servo.cxx

```diff
@@ -0,0 +1,88 @@
+/*
+ *  Servo.cxx
+ *  Copyright 2020 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include <Arduino.h>
+#include "BoardDataDef.hxx"
+#include "Servo.h"
+
+#define SERVO_MIN() (MIN_PULSE_WIDTH - this->min * 4)
+#define SERVO_MAX() (MAX_PULSE_WIDTH - this->max * 4)
+
+#define TRIM_DURATION 2
+
+uint32_t servoCount = 0;
+
+Servo::Servo() {
+    if (servoCount < board_data->servo_value.max_size()) {
```

Comment:
Clearly you have made no research as to what constant `std::vector<T>::max_size()` returns